__mocks__/@zextras/carbonio-shell-ui.tsx

```diff
@@ -23,6 +23,12 @@ export const USER_SETTINGS: AccountSettings = {
 	}
 };
 
+const getNode = jest.fn(() => Promise.resolve({ name: 'root', id: 'ROOT_ID' }));
+const getNodeAvailable = true;
```

Comment:
To keep all clear these mocks should be put under a carbonio-files-ui.tsx and then imported here. So with all the future usage of integrations from other modules it will be more ordered.